### DISCUSSION TITLE
sigma.lv

### DISCUSSION THREAD

**Discussion by AlainZuur** [JenniNiku/gllvm#143] [General]
Hello,

This is about the sigma.lv. Can I just double-check my understanding of this?

I understand that in a concurrent ordination, these sigmas determine the variation of the unconstrained LV. That is clear.

But what exactly is their role in an ordinary GLLVM (model based ordination)? I understand that the upper diagonal elements of the thetas are set to 0. Is the reason for the construction:

M1$params$theta %*% diag(M1$params$sigma.lv)

to enforce the diagonal elements of theta to be positive? In other words....you set the diagonal elements of theta to 1 during the estimation process, and estimate the sigma.lv somehow (??)...and by post-multiplication you have managed to avoid rotation invariance and ensured parameter identifiability?

In other words...the sigma.lv plays a different role in model-based ordination and RRR/concurrent ordination?

Kind regards,

Alain

**Comment by BertvanderVeen** [comment]
Yes, they indeed play a different role in unconstrained/residual ordination. In RRR they are not used, in concurrent ordination they are the variance of the LV-level errors, in unconstrained ordination they `theta%*%diag(sigma.lv)` is only a reparameterization of the original loadings. Identifiability constraints could have been chosen differently, but this was the most compatible choice with the existing methods in `gllvm` when I implemented the RRR and concurrent ordination code.

Previously they were simply estimated as part of `theta`, but they are now separated purely for compatibility with RRR and concurrent ordination. This does not affect the ordination in any way, since the model is equivalent up to the reparameterization, and the loadings are `theta%*%diag(sigma.lv)`, where the main diagonal elements of theta are fixed so that sigma.lv is identified.

**Reply by AlainZuur** [reply]
Thanks....then I can tick that off.

Alain
